README.md

```diff
@@ -1 +1,164 @@
-# afctl
\ No newline at end of file
+# afctl
+
+The proposed CLI tool is being authored to make creating and deployment of airflow projects faster and smoother. 
+As of now, there is no tool out there that can empower the user to create a boilerplate code structure for airflow 
+projects and make development + deployment of projects seamless.
+
+## Installation
+
+Clone the repository. <br />
+Create a new python virtualenv. You can use the following command. <br />
+```bash
+python3 -m venv <name>
+```
+Activate your virtualenv<br/>
+```bash
+source /path_to_venv/bin/activate
+```
+
+Step into the afctl directory and run <br/>
+```bash
+pip3 install .
+```
+
+## Requirements
+Python 3.5+
+
+## Usage
+
+Commands right now supported are
+* init
```

Comment:
new or init? @chattarajoy  @lordozb

Reply:
I kept init as it was more intuitive. "Initialise the project" because not always you'll require the directories to be generated. (if cloned an afctl project from github)